stats/stats/src/update/total_blocks.rs

```diff
@@ -0,0 +1,276 @@
+use async_trait::async_trait;
+use blockscout_db::entity::blocks;
+use chrono::NaiveDateTime;
+use entity::{chart_data_int, charts};
+use sea_orm::{
+    sea_query, ColumnTrait, DatabaseConnection, EntityTrait, FromQueryResult, QueryFilter,
+    QueryOrder, QuerySelect, Set,
+};
+
+use super::UpdateError;
+
+#[derive(FromQueryResult)]
+struct TotalBlocksData {
+    number: i64,
+    timestamp: NaiveDateTime,
+}
+
+#[derive(Debug, FromQueryResult)]
+struct ChartId {
+    id: i32,
+}
+
+#[derive(Default, Debug)]
+pub struct Updater {}
+
+#[async_trait]
+impl super::UpdaterTrait for Updater {
+    async fn update(
+        &self,
+        db: &DatabaseConnection,
+        blockscout: &DatabaseConnection,
+    ) -> Result<(), UpdateError> {
+        let name = "totalBlocksAllTime";
+        let data = blocks::Entity::find()
+            .column(blocks::Column::Number)
+            .column(blocks::Column::Timestamp)
+            .order_by_desc(blocks::Column::Timestamp)
```

Comment:
I think it's better to order by block number. 

We are absolutely sure that blocks have to be sorted by block number, but ordering by timestamp is more tricky and maybe in some blockchains it's not always true.